source/game/kart/KartParamFileManager.cc

```diff
@@ -107,8 +105,7 @@ EGG::RamStream KartParamFileManager::getBikeDispParamsStream(Vehicle vehicle) co
     auto *file = reinterpret_cast<ParamFile<KartParam::BikeDisp> *>(m_bikeDispParam.file);
     ASSERT(file);
     void *offset = &file->params[idx];
-    u32 size = sizeof(KartParam::BikeDisp);
-    return EGG::RamStream(reinterpret_cast<u8 *>(offset), size);
+    return EGG::RamStream(offset, sizeof(KartParam::BikeDisp));
```

Comment:
This can be improved further by sending in `&file->params[idx]` instead of `offset`.